pkg/yamlmeta/internal/yaml.v2/decode_test.go

```diff
@@ -700,7 +700,7 @@ var unmarshalTests = []struct {
 		M{"a": 123456e1},
 	}, {
 		"a: 123456E1\n",
-		M{"a": 123456E1},
+		M{"a": 123456e1},
```

Comment:
is this go version difference, or something else?

Reply:
We think this might be from a dependency bump that did not get fully updated. You can see this happening on develop after running `./hack/build.sh` with `go version go1.16.8 darwin/amd64`.

Reply:
I suspect this slipped in on accident. It's related to the floating-point fix from a prior commit.

In Go, either case is permitted (i.e. `e` vs. `E`).

Also, we're explicitly excluding these tests in `ytt`'s suite.

I recommend that until we bring `k14s/yaml` fully up to the current state of `ytt`'s copy of that library, let's make no further changes to these files (i.e. remove this particular change).